appservice/src/tree/DeploymentTreeItem.ts

```diff
@@ -132,4 +133,14 @@ export class DeploymentTreeItem extends AzureTreeItem<ISiteTreeRoot> {
             return '';
         }
     }
+
+    private removeLineBreaks(message: string): string {
```

Comment:
nit: `removeLineBreaks` makes it sound like you're just flattening the message into one line. Suggest renaming to something like `getFirstLine`